nuxeo-distribution/nuxeo-server-tests/src/test/java/org/nuxeo/ftest/server/ITDevHotReloadTest.java

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
would be interesting to have a similar test on the jsf-ui, usable by studio integration tests on 9.x